docs/types.md

```diff
@@ -1426,6 +1427,23 @@ Aliases: [`Standardized`](#standardized), [`ModuleSpecifier`](#modulespecifier)
 
 ---
 
+#### importExpression
+
+```js title="JavaScript"
+t.importExpression(source, options);
+```
+
+See also `t.isImportExpression(node, opts)` and `t.assertImportExpression(node, opts)`.
+
+AST Node `ImportExpression` shape:
+- `source`: `Expression` (required)
+- `options`: `Expression` (default: `null`)
+- `phase`: `"source" | "defer"` (default: `null`, excluded from builder function)
```

Comment:
Is this expected?

Reply:
Should we add it to the builder parameter? (excluded from builder function)

Reply:
The source phase import is a stage-3 proposal, we can wait till it gets advanced. Note that it is not uncommon that builder function excludes some AST properties, for example `decorators` are also excluded from `ClassMethod`'s builder. We could add them to the builder when the decorator feature is finished.